### ISSUE TITLE
Select first item by default in open/tabopen/etc

### ISSUE THREAD

**Issue by bpmcl** [tridactyl/tridactyl#1370]
Recently migrated from Vimium, and really liking Tridactyl so far, but I have one small configuration option I would like to suggest.

Most of the time when I am using functions like `open` and `tabopen`, I know exactly what I am looking for, so I can type a couple of letters and the first item in the list is what I wanted. Rather than having to hit Tab to select it, an option to have it selected by default would be useful, instead of always performing a web search by default.

(Additionally, being able to navigate up and down the list with the arrow keys would be nice.)

**Comment by bovine3dom** [comment]
Up and down you can do - `bind --mode=ex <Tab>` will tell you what to bind to, then `bind --mode=ex <[...I can't remember what the up arrow is called, maybe ArrowUp... It's listed on MDN somewhere]> [the thing you just found]`. Repeat for `<S-Tab>` and whatever the down arrow is called.

We can definitely do the first one - that's how buffer behaves - but I think it would annoy most people as space inserts the highlighted completion, so you wouldn't be able to do `tabopen why do people use emacs`.

**Comment by bpmcl** [comment]
I don't think it should be the default setting for everyone, but a configuration option to allow it to be customised would be useful.

Thanks for the help on the arrow key binding 👍

**Comment by bovine3dom** [comment]
No it's not possible yet - we have `:set completions.History.autoselect true` _but_ the code for that isn't in History completions yet. It works for bookmarks, tabs, RSS and undo completions though.

It would not be hard to add.

https://github.com/tridactyl/tridactyl/blob/ced62d5fca6ea5aa986975cf5cf1765844498363/src/completions/Rss.ts#L35

something like that should go in History.ts

**Comment by stevium** [comment]
@bovine3dom On the other hand, is there a way to show *currently opened* tab as selected one when typing `:tab` ? I almost always find myself looking for adjacent tabs to the currently opened one, and being able to press `ArrowUp` or `ArrowDown` to get to the start or end of the list doesn't really help much :)

**Comment by stevium** [comment]
Oh wow, that was exactly what I was looking for! Works like a charm!
